src/ztc/api/serializers/zaken.py

```diff
@@ -130,13 +131,23 @@ class ZaakTypeSerializer(
         resource=Eigenschap,
         resource_serializer=EigenschapSerializer,
     )
+
     zaakobjecttypen = HistoryURLField(
         method_name="create_custom_urls",
         resource=ZaakObjectType,
         resource_serializer=ZaakObjectTypeSerializer,
     )
 
+    @extend_schema_field(
+        {
+            "type": "array",
+            "items": {"type": "string"},
+            "description": "description set in vng-api-common/inspector/view/_map_serializer_field",
+            "example": ["http://example.com"],
+        },
+    )
```

Comment:
Is dit nodig? Het lijkt mij iedergeval niet de juiste plek omdat `create_custom_urls` (hetgene waar de decorator op werkt) geen veld is.

Reply:
https://drf-spectacular.readthedocs.io/en/latest/customization.html?highlight=extend_schema_field#step-3-extend-schema-field-and-type-hints

zie step 3

"You can apply it also to the method of a SerializerMethodField."

Dit is nodig omdat drf-spectacular uitzichzelf niet snapt wat dit veld is / doet.

Reply:
als je hem in de HistoryURLFIELD zet bijvoorbeeld krijg je deze warning:

Warning #3: ZaakTypeViewSet: ZaakTypeSerializer: unable to resolve type hint for function "create_custom_urls". Consider using a type hint or @extend_schema_field. Defaulting to string.

Reply:
een type hint is doorgaans een stuk eenvoudiger dan :)

```py
def create_custom_urls(self, zaaktype, resource, resource_serializer) -> List[str]:
    ...
```

Reply:
Ah oke bedankt @sergei-maertens, dit lijkt mij ook makkelijker.